src/mame/drivers/firebeat.cpp

```diff
@@ -136,13 +136,16 @@
         it appears to be a 4-dword FIFO or something along those lines.
 */
 
+#include <cmath>
+
```

Comment:
Please include standard library headers includes after all the MAME headers (this would go between wdlfft/fft.h and firebeat.lh).